uwblueprint/childrens-aid-society
# [kai/db-seed-script] Set up database seed script

## Notion ticket link

[Set up database seed script](https://www.notion.so/uwblueprintexecs/134e1f159caa4f73939d11a700c325ad?v=39cf7d5a98384eb9af2bff2265d0ac8a&p=a541358c29f44c6d86caf516088872e1&pm=s)



## Implementation description
* Add script to populate dev db with mock data
* Add db nuke script for convenience



## Steps to test
1. `docker-compose down --volumes`
2. `docker-compose up`
3. `docker exec -it cas_py_backend /bin/bash -c "python -m tools.db_seed"`
4. `docker exec -it cas_db /bin/bash`
5. `psql -U postgres -d cas`
6. `\d`
7. `select * from children` and other tables as necessary
8. `docker exec -it cas_py_backend /bin/bash -c "python -m tools.db_seed nuke"`
9. enter the db again and see what is there



## What should reviewers focus on?
- cleanliness of code in `backend/python/tools/db_seed.py` (i am new to sqlalchemy)
- readability of code (i.e. 80 char line limit)


## Checklist
